extensions/amp-social-share/0.1/amp-social-share.css

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import "../../../third_party/optimized-svg-icons/amp-social-share-svgs.css";
+
+.amp-social-share {
+}
+
+.-amp-social-share {
+  display: block;
+  border-radius: 5px;
+}
+
+.-amp-social-share a {
+  display: block;
+  padding: 12px;
+  background-repeat: no-repeat;
+  background-position: center;
+  text-decoration: none !important;
+}
+
+/* Twitter Styling */
+.-amp-social-share-twitter {
```

Comment:
Don't you want these overridable? In that case something like 
`amp-social-share .twitter` seems better